service-api/src/main/java/greencity/dto/habit/AddCustomHabitDtoResponse.java

```diff
@@ -0,0 +1,29 @@
+package greencity.dto.habit;
+
+import greencity.constant.ServiceValidationConstants;
+import greencity.dto.habittranslation.HabitTranslationDto;
+import greencity.dto.shoppinglistitem.CustomShoppingListItemResponseDto;
+import lombok.*;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import java.util.List;
+import java.util.Set;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Getter
+@Setter
+@EqualsAndHashCode
+public class AddCustomHabitDtoResponse {
+    private Long id;
+    private Long userId;
+    @Min(value = 1, message = ServiceValidationConstants.HABIT_COMPLEXITY)
+    @Max(value = 3, message = ServiceValidationConstants.HABIT_COMPLEXITY)
+    private String image;
```

Comment:
it seems like this validation is not related to the `image`

Reply:
Yes, it should be for the "complexity" field. There is a mistake.

Reply:
If you had tests for validation it would be easy to catch it
It is a good example of why tests are important

Reply:
added